.github/workflows/merge-main.yml

```diff
@@ -205,54 +205,34 @@ jobs:
       - codeql
       - sonarcloud
       - release
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
     environment:
       name: test
     env:
       ZONE: test
       NR_SPAR_BACKEND_VERSION: ${{ needs.release.outputs.version }}
+    strategy:
+      matrix:
+        name: [ backend ]
```

Comment:
For TEST and PROD include the init and database deployment, using `overwrite: false` to make sure they don't replace exciting deployments.

Here's an example from the QuickStart:

```
 deploys:
    name: Deploys
    needs:
      - builds
    runs-on: ubuntu-22.04
    strategy:
      matrix:
        name: [ init, database, backend ]
        include:
          - name: init
            file: common/openshift.init.yml
            overwrite: false
          - name: database
            file: database/openshift.deploy.yml
            overwrite: false
          - name: backend
            file: backend/openshift.deploy.yml
            overwrite: true
            parameters: -p MIN_REPLICAS=1 -p MAX_REPLICAS=2 -p BUILD=snapshot-${{ github.event.number }}
    steps:
      - uses: bcgov-nr/action-deployer-openshift@v1.0.1
        with:
          file: ${{ matrix.file }}
          oc_namespace: ${{ secrets.OC_NAMESPACE }}
          oc_server: ${{ secrets.OC_SERVER }}
          oc_token: ${{ secrets.OC_TOKEN }}
          overwrite: ${{ matrix.overwrite }}
          penetration_test: false
          verification_path: "actuator/health"
          parameters:
            -p ZONE=${{ github.event.number }} -p NAME=${{ github.event.repository.name }}
            -p PROMOTE=${{ github.repository }}/${{ matrix.name }}:${{ github.event.number }}
            ${{ matrix.parameters }}
```